frontend/src/upload_file.ts

```diff
@@ -155,7 +155,30 @@ class UploadFile {
     files.forEach(file => {
       const { fieldName, formId, renderer, uploads, uploadUrl } = this;
       const filename = file.name;
-      const uploadIndex = uploads.length;
+      let uploadIndex = uploads.length;
+
+      // #323 remove existing file
+      for (let index = 0; index < this.uploads.length; ++index) {
+        let existing_upload = this.uploads[index];
```

Comment:
I get an `eslint` warning here. It can be defined as a `const`. 
I would use camelCase: `existingUpload`

Reply:
done